website/src/app/kb/deploy/gateways/readme.mdx

```diff
@@ -31,31 +37,60 @@ you'll need to make sure the following outbound traffic is allowed:
+-- in some cases, even on the same physical host. This ensures the lowest
+possible latency and highest possible throughput for client connections, and
+allows you to more easily deploy additional gateways for a particular resource
+that needs to handle more client connections.
+
+The downside of deploying gateways close to resources is that it increases the
+number of overall gateways you need to deploy, possibly making it more difficult
+to manage and monitor them. Firezone's control plane is designed to make this as
+easy as possible, but it's still something to keep in mind when planning your
+deployment.
+
+### Sizing recommendations
 
 Gateways, like the rest of Firezone's data plane stack, are written in Rust and
-are thus resource efficient by nature. A single gateway running on a 2 vCPU VM
-with 1 GB of memory should be able to handle hundreds of client connections
-under typical usage patterns.
+are thus resource efficient by nature.
 
-Network throughput is usually constrained by single-thread performance due to
-the WireGuard state machine and overhead in the kernel network stack. Still,
-most gateways should be able to saturate 1 Gbps with relative ease. Faster
-speeds can be achieved with syscall optimization and multi-threaded
-optimizations, both of which are currently being worked on.
+A single client connection to a single gateway can typically reach speeds of **1
+Gbps** or more. This scales linearly with each client connection up to the
+number of CPU cores available to the Gateway.
+
+Use the table below as a rough guide for sizing your gateway deployments:
+
+| Gateway size | Users served | CPU cores | Memory | Network link |
+| ------------ | ------------ | --------- | ------ | ------------ |
+| Small        | 100          | 2         | 2 GB   | 1 Gbps       |
+| Medium       | 1,000        | 8         | 8 GB   | 10 Gbps      |
+| Large        | 10,000       | 32        | 32 GB  | 40 Gbps      |
+
+To go beyond the table above, you can deploy additional Gateways and use
+Firezone's [automatic load balancing](#load-balancing) to distribute client
+connections across them.
 
 ## Deploy a single gateway
 
 Deploying a single gateway can be accomplished in the admin portal.
 
-Go to `Sites` -> `<site>` -> `Deploy a Gateway` and follow the prompts to deploy
-for your preferred environment. This will deploy a single gateway.
+Go to `Sites` -> `<name of site>` -> `Deploy a Gateway` and follow the prompts
+to deploy for your preferred environment. This will deploy a single gateway.
 
 See the [upgrading guide](/kb/administer/upgrading) for information on keeping
 your gateway up-to-date.
 
 ## Deploy multiple gateways
 
+To achieve [high availability](#high-availability), deploy more gateways within
+the same site.
+
+You can deploy multiple gateways from the admin portal, or automate the process
+by reusing the `FIREZONE_TOKEN` environment variable shown when deploying a
+gateway from the admin portal.
```

Comment:
```suggestion
gateway from the admin portal. All gateways within the sate **must** operate in the same network environment.
```

Reply:
This is mentioned more explicitly in the Sites documentation and at the top of the deploy docs